src/Microsoft.AspNet.Server.Kestrel/Http/FrameResponseStream.cs

```diff
@@ -23,13 +23,7 @@ public FrameResponseStream(FrameContext context)
 
         public override bool CanWrite => true;
 
-        public override long Length
-        {
-            get
-            {
-                throw new NotImplementedException();
-            }
-        }
+        public override long Length => Position;
```

Comment:
No it won't, and it won't work for Nowin either. https://github.com/aspnet/Hosting/pull/403#discussion_r41928042